functions/src/db/getUserToken.ts

```diff
@@ -0,0 +1,26 @@
+import { UID } from '../types/uid'
+import { JWTToken } from '../types/JWT'
+import { db } from '../firebase/firebase'
+import { UserData } from '../types/UserData'
+
+export const getUserToken = async (userId: UID): Promise<JWTToken | null> => {
+    const userDocumentSnapshot = await db.collection('users').doc(userId).get()
+
+    if (!userDocumentSnapshot?.exists) {
+        return null
```

Comment:
for every created user, a token should normally exist. If it does not, I would say it is an error case